indexing-service/src/main/java/org/apache/druid/indexing/overlord/DruidOverlord.java

```diff
@@ -155,7 +173,6 @@ public void stop()
                   serviceAnnouncer.unannounce(node);
                   compactionScheduler.stop();
                   taskMaster.stopBeingLeader();
```

Comment:
won't this cause the overlord to stop being the half/task leader before the supervisormanager has stopped? since lifecycle.stop is executed in reverse order

Reply:
Changed this to `downgradeToHalfLeader()`.